test/utils/templates.spec.ts

```diff
@@ -16,10 +16,20 @@ describe('Templates Utils', () => {
 
   it('should return information which files to copy where', () => {
     const initCopyInfo = getCopyDescriptors('targetDir', getTemplatePaths(['init']));
-    expect(initCopyInfo.map(copyInfo => path.basename(copyInfo.fileName)).sort()).toMatchSnapshot();
+    expect(
+      initCopyInfo
+        .map(copyInfo => copyInfo.fileName.split('targetDir' + path.sep)[1])
+        .sort()
+        .map(fileName => fileName.split(path.sep))
```

Comment:
This should be moved to a function that explains what it does:

``` ts
copyInfoToPathArray(copyInfo: string[]): Array<string[]> { // feel free to use a better name
  return paths
    .map(copyInfo => copyInfo.fileName.relative(path.resolve('targetDir'), path))
    .sort()
    .map(filePath => filePath.split(path.sep));
}
```

Reply:
@florian-richter done and tests are green